exercise-1/style.css

```diff
@@ -0,0 +1,149 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  text-decoration: none;
+  list-style: none;
+}
+
+html,
+body {
+  height: 100vh;
+  margin: 0;
+  font-family: 'Lato', 'san-serif';
+}
+
+button {
```

Comment:
It is generally good practice to avoid using element selectors. Prefer e.g. classes instead. The reasoning is simple: element selector are pretty intrusive (i.e. they update the styles of each matching element), and therefore you will quickly end up with having to introduce styles to override (undo) the settings in these rules.